scss/typography/_helpers.scss

```diff
@@ -111,12 +104,6 @@ $dropcaps-margin: 0.192em 0.0962em 0 0 !default;
   
   // Use to create an initial aka dropcap
   .dropcaps{
-    line-height: 1.5; // Disabling default line height of an element
-    &:first-letter {
-      @include foundation-dropcaps;
-    }
+     @include dropcaps();
   }
-}
-
-
-
+}
```

Comment:
Yes. Always let one end-of-line (not an empty line) at the end of files. If you use an editor like Sublime Text or Atom, there is plugin to do that automatically